app/assets/javascripts/shared/datatable/state.js

```diff
@@ -0,0 +1,71 @@
+DradisDatatable.prototype.setupStateButtons = function() {
+  if (this.$paths.data('table-state-url') === undefined){
+    return [];
+  }
+
+  var states = [['Draft', 'fa-pencil-square-o'], ['Published', 'fa-rocket']],
+    stateButtons = [];
+
+  states.forEach(function(state){
+    stateButtons.push({
+      text: $(`<i class="fa ${state[1]} fa-fw"></i><span>${state[0]}</span>`),
+      action: this.updateIssueState(state[0])
+    });
+    }.bind(this)
+  );
+
+  return stateButtons;
+};
+
+DradisDatatable.prototype.updateIssueState = function(newState) {
+  return function() {
+    var that = this;
+    var selectedRows = this.dataTable.rows({ selected: true });
+
+    $.ajax({
+      url: this.$paths.data('table-state-url'),
+      method: 'PUT',
+      data: { ids: that.rowIds(selectedRows), state: newState },
+      beforeSend: function (){
+        that.toggleLoadingState(selectedRows, true);
+      },
+      success: function(){
+        that.toggleLoadingState(selectedRows, false);
+        that.toggleStateBtn(false);
+
+        selectedRows.deselect().remove().draw();
+
+        $('[data-behavior="qa-alert"]').remove();
+        $('.page-title').after(`
+          <div class="alert alert-success" data-behavior="qa-alert">
+            <a class="close" data-dismiss="alert" href="javascript:void(0)">x</a>
+            Successfully set the issues as ${newState}!
```

Comment:
```suggestion
            <p>Successfully set the issues as ${newState}!</p>
```

We should have text directly in a div

Reply:
Looks like this when I add the paragraph tag:

<img width="383" alt="Screenshot 2023-03-10 at 6 42 39 PM" src="https://user-images.githubusercontent.com/1908272/224295695-d31e79e4-9255-4fc0-8c58-222f05ba8cc2.png">

Note that the text is inside the `<div class="alert">` and this is the same structure we use for `flash_messages` and alerts from the datatabel (delete.js).